src/templates/admin/production/edit_typesetter_assignment.html

```diff
@@ -73,6 +74,14 @@ <h2>Edit Assignment</h2>
                     <textarea cols="40" id="id_typeset_task" name="typeset_task" rows="10"
                               data-origin="textarea">{{ typeset.typeset_task }}</textarea>
                     <button class="button success" type="submit" name="update"><i class="fa fa-check">&nbsp;</i>Update Task</button>
+                    {% elif typeset.status.slug == 'declined' %}
+                    <p>This typesetting task has been declined. If this was done in error you can
+                        reset it here.</p>
+                        <button class="button" name="reset"><i class="fa fa-recycle"></i> Reset</button>
+
+                    {% else %}
+                    <p>This typesetting task has been accepted so cannot be edited.</p>
+                    {% endif %}
```

Comment:
I think this is equivalent and easier to read:
```
{% if typeset.accepted %}
[...]
{% elif typeset.declined %}
[...]
{% else %}
[...]
{% endif %}
```
I'm assuming that a task can't be both accepted and declined at the same time.